src/test/scala/mrpowers/jodie/DeltaHelperSpec.scala

```diff
@@ -729,4 +729,51 @@ class DeltaHelperSpec
         orderedComparison = false)
     }
   }
+  describe("Remove rows from DeltaTable where rows exists in a Source Dataframe") {
```

Comment:
please add unit test of failure scenarios as well, for example what would happen if the operator send is invalid or the column does not exists in the table, etc.